tests/bson/bson-utcdatetime-001.phpt

```diff
@@ -51,8 +51,8 @@ Test#1 { "0" : { "$date" : 1416445411987 } }
 string(37) "{ "0" : { "$date" : 1416445411987 } }"
 string(37) "{ "0" : { "$date" : 1416445411987 } }"
 bool(true)
-Test#2 { "0" : {  } }
-string(14) "{ "0" : {  } }"
-string(14) "{ "0" : {  } }"
+Test#2 { "0" : { "date" : "2014-11-20 01:03:31.987%r(0+)%r", "timezone_type" : 1, "timezone" : "+00:00" } }
```

Comment:
I removed DateTime from the *new* tests, but planned to leave it as-is here. AFAICT, it does have undocumented public properties. `var_dump(new DateTime);` produces:

```
class DateTime#1 (3) {
  public $date =>
  string(26) "2015-09-15 12:50:49.000000"
  public $timezone_type =>
  int(3)
  public $timezone =>
  string(16) "America/New_York"
}
```